core/src/test/java/org/apache/iceberg/actions/TestBinPackStrategy.java

```diff
@@ -138,6 +138,43 @@ public void testGroupingMinInputFilesInvalid() {
         0, Iterables.size(grouped));
   }
 
+  @Test
+  public void testGroupingMinInputFilesAsOne() {
+    RewriteStrategy strategy = defaultBinPack().options(ImmutableMap.of(
+            BinPackStrategy.MIN_INPUT_FILES, Integer.toString(1),
+            BinPackStrategy.MAX_FILE_SIZE_BYTES, Long.toString(3 * MB),
+            RewriteDataFiles.TARGET_FILE_SIZE_BYTES, Long.toString(2 * MB),
+            BinPackStrategy.MIN_FILE_SIZE_BYTES, Long.toString(MB),
+            BinPackStrategy.DELETE_FILE_THRESHOLD, Integer.toString(2)
+    ));
+
+    // single file within a group, with size within threshold and no deletes should be considered as NOOP.
```

Comment:
I think we are aiming to not add concluding punctuation on comments. So I would keep it consistent with your other comments in the file and drop the final period.